apps/SageAccountWeb/src/LoginPage.tsx

```diff
@@ -48,6 +82,14 @@ function LoginPage(props: LoginPageProps) {
             <Box sx={{ my: 4 }}>
               <StandaloneLoginForm
                 sessionCallback={() => {
+                  if (sourceApp?.friendlyName && sourceApp.appURL) {
+                    setLastLoginInfo(
+                      getLoginMethod(history),
+                      formatDate(new Date()),
```

Comment:
Good call! Updated to store the ISO timestamp, format the date on render, and use `window` instead of `history. 

It looks like `useSourceApp` and `AppInitializer` don't handle unknown `appIds` gracefully -- I opened [this ticket](https://sagebionetworks.jira.com/browse/PORTALS-2746) to track handling this case.